src/ReferenceTrimmer.csproj

```diff
@@ -22,8 +23,20 @@
       <Pack>true</Pack>
       <PackagePath>buildMultiTargeting\</PackagePath>
     </None>
+    <None Update="tools\*.ps1">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Pack>true</Pack>
+      <PackagePath>tools\</PackagePath>
+    </None>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\analyzer\ReferenceTrimmerAnalyzer.csproj" ReferenceOutputAssembly="false" />
     <InternalsVisibleTo Include="ReferenceTrimmer.Tests" />
   </ItemGroup>
+
+  <Target Name="_AddAnalyzersToOutput">
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="..\analyzer\$(IntermediateOutputPath)ReferenceTrimmerAnalyzer.dll" PackagePath="analyzers/dotnet/cs" />
```

Comment:
Using `IntermediateOutputPath` isn't a great pattern since that's *this* project's property, not the referenced project's.

Instead you can use `OutputItemType` to assign the target path to a different item than "Reference".